snow/engine/common/test_engine.go

```diff
@@ -354,3 +357,8 @@ func (e *EngineTest) Chits(validatorID ids.ShortID, requestID uint32, containerI
 	}
 	return nil
 }
+
+// IsBootstrapped ...
+func (e *EngineTest) IsBootstrapped() bool {
+	return e.Bootstrapped
+}
```

Comment:
The rest of the test engine allows for specifying function callbacks. It should be handled the same way here.